thinc/layers/array_getitem.py

```diff
@@ -1,9 +1,14 @@
+import sys
 from typing import Union, Sequence, Tuple, TypeVar
 from ..types import ArrayXd, FloatsXd, IntsXd
 from ..model import Model
```

Comment:
```suggestion
from typing import Union, Sequence, Tuple, TypeVar
from ..types import ArrayXd, FloatsXd, IntsXd
from ..model import Model

import sys
```
(you couldn't know this, but this order is more in line with other files in the codebase. We typically have the `from typing` imports on the first line(s))

Reply:
Removing the `import sys` statement because it's not needed since we're handling exposing `SupportsIndex` in `thinc.types.